portfolio/src/main/webapp/script.js

```diff
@@ -43,16 +53,15 @@ function addRandomQuote() {
  */
 function randomizeImage() {
   const imageIndex = Math.floor(Math.random() * 18) + 1;
-  const imgUrl = 'images/cats/cats-' + imageIndex + '.jpg';
-
-  const imgElement = document.createElement('img');
-  imgElement.src = imgUrl;
-  imgElement.setAttribute('height', '500');
+  const imageUrl = 'images/cats/cats-' + imageIndex + '.jpg';
```

Comment:
Use template literals over complex string concat: https://google.github.io/styleguide/jsguide.html#features-strings-template-strings

Here you can write sth. like const imageUrl = `images/cats/cats-${imageIndex}.jpg`